main.go

```diff
@@ -106,8 +113,7 @@ func PrepareCmd() *cobra.Command {
 			occ.PrepareOpenMetadataForFybrik(endpoint, user, password, customization, &logger)
 		},
 	}
-	cmd.Flags().StringVar(&customizationFile, "customization", customizationFile,
-		"File containing tags and custom properties needed for working with Fybrik")
+	cmd.Flags().StringVar(&customizationFile, Customization, customizationFile, FileContainingTagsAndPropertiesNeeded)
```

Comment:
why do you print customizationFile twice  ?

Reply:
`&customizationFile` is where to place the value, whereas `customizationFile` is the default value.